### ISSUE TITLE
Compatibility Issue

### ISSUE THREAD

**Issue by Theroh** [GC-spigot/AdvancedEnchantments#865]
**Describe the bug**
In my server I'm currently using Slime Fun and AE. Althought a problem appeared yesterday (updating Slime Fun), everytime a user uses one item from slimefun the item goes to the player head replacing any head gear they have and sometimes breaking the item. We tried to isolate the problem with any head related plugins in a test server, and happens that the plugin that caused the compatibility bug was Advanced Enchantments.

**To Reproduce**
Simply put slimefun 4 and the latest version os AE
(https://thebusybiscuit.github.io/builds/TheBusyBiscuit/Slimefun4/master/)

**Screenshots / Video**
https://www.youtube.com/watch?time_continue=44&v=2kayVizmCt4&feature=emb_logo

**Server Information (please complete the following information):**
- PaperSpigot 1.15.2
- Plugin Version 7.0.3

**Comment by GC-spigot** [comment]
Set this to false: https://prnt.sc/rdvg0t
